foxit-reader@10.0.0.35798: download via aria2 failed

I can not install foxit reader

PS C:\Users\migue> scoop install foxit-reader
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Installing 'foxit-reader' (10.0.0.35798) [64bit]
Starting download with aria2 ...
Download: [#40d690 0B/0B CN:1 DL:0B]
Download: [#40d690 0B/0B CN:1 DL:0B]
Download: 05/08 19:54:42 [ERROR] CUID#8 - Download aborted. URI=https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.0.0/B482F50BE0AE94322AED580B59D2C060/FoxitReader1000_enu_Setup_Prom.exe
Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.0.0/B482F50BE0AE94322AED580B59D2C060/FoxitReader1000_enu_Setup_Prom.exe
Download:   -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 40d690|ERR |       0B/s|C:/Users/migue/scoop/cache/foxit-reader#10.0.0.35798#https_cdn01.foxitsoftware.com_product_reader_desktop_win_10.0.0_B482F50BE0AE94322AED580B59D2C060_FoxitReader1000_enu_Setup_Prom.exe
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.
ERROR Download failed! (Error 3) Resource was not found
ERROR https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.0.0/B482F50BE0AE94322AED580B59D2C060/FoxitReader1000_enu_Setup_Prom.exe
    referer=https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.0.0/B482F50BE0AE94322AED580B59D2C060/
    dir=C:\Users\migue\scoop\cache
    out=foxit-reader#10.0.0.35798#https_cdn01.foxitsoftware.com_product_reader_desktop_win_10.0.0_B482F50BE0AE94322AED580B59D2C060_FoxitReader1000_enu_Setup_Prom.exe

ERROR & 'C:\Users\migue\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\migue\scoop\cache\foxit-reader.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=7684 --continue

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/lukesampson/scoop-extras/issues/new?title=foxit-reader%4010.0.0.35798%3a+download+via+aria2+failed

--- a/bucket/foxit-reader.json
+++ b/bucket/foxit-reader.json
@@ -6,7 +6,7 @@
         "identifier": "Freeware",
         "url": "https://www.foxitsoftware.com/pdf-reader/eula.html"
     },
-    "url": "https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.0.0/B482F50BE0AE94322AED580B59D2C060/FoxitReader1000_enu_Setup_Prom.exe",
+    "url": "https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.0/B482F50BE0AE94322AED580B59D2C060/FoxitReader100_enu_Setup_Prom.exe",
     "hash": "md5:b482f50be0ae94322aed580b59d2c060",
     "bin": "FoxitReader.exe",
     "shortcuts": [
@@ -18,10 +18,10 @@
     "innosetup": true,
     "checkver": {
         "url": "https://www.foxitsoftware.com/downloads/downloadForm.php?retJson=1&product=Foxit-Reader&platform=Windows&language=English",
-        "regex": "([\\d.]+).*(?<md5hash>([a-fA-F0-9]{32}))"
+        "regex": "([\\d.]+).*\\\\/(?<dlver>[\\d\\.]+)\\\\/(?<md5hash>([a-fA-F0-9]{32}))\\\\/(?<dlname>.+\\.exe)"
     },
     "autoupdate": {
-        "url": "https://cdn01.foxitsoftware.com/product/reader/desktop/win/$majorVersion.$minorVersion.$patchVersion/$matchMd5hash/FoxitReader$majorVersion$minorVersion$patchVersion_enu_Setup_Prom.exe",
+        "url": "https://cdn01.foxitsoftware.com/product/reader/desktop/win/$matchDlver/$matchMd5hash/$matchDlname",
         "hash": {
             "url": "https://www.foxitsoftware.com/downloads/downloadForm.php?retJson=1&product=Foxit-Reader&platform=Windows&language=English",
             "regex": "$md5"

--- a/bucket/foxit-reader.json
+++ b/bucket/foxit-reader.json
@@ -1,13 +1,20 @@
 {
     "version": "10.0.0.35798",
-    "homepage": "https://www.foxitsoftware.com/",
     "description": "Super-fast, feature rich PDF reader that offers a delightful reading experience in a small footprint.",
+    "homepage": "https://www.foxitsoftware.com/",
     "license": {
         "identifier": "Freeware",
         "url": "https://www.foxitsoftware.com/pdf-reader/eula.html"
     },
-    "url": "https://cdn01.foxitsoftware.com//pub/foxit/reader/desktop/win/10.x/10.0/en_us/FoxitReader100_Setup_Prom_IS.exe",
+    "url": "https://cdn01.foxitsoftware.com/pub/foxit/reader/desktop/win/10.x/10.0/en_us/FoxitReader100_Setup_Prom_IS.exe#/dl.7z",
     "hash": "eac0bf3cdb0b629381c61fe9c1767d310f8654e34e30f19145d3e88f816cac00",
+    "extract_dir": "$PLUGINSDIR",
+    "installer": {
+        "script": [
+            "Get-ChildItem \"$dir\" -Exclude 'Foxit*Prom.exe' | Remove-Item",
+            "Get-Item \"$dir\\Foxit*prom.exe\" | Expand-InnoArchive -Destination \"$dir\" -Removal"
+        ]
+    },
     "bin": "FoxitReader.exe",
     "shortcuts": [
         [
@@ -15,13 +22,11 @@
             "Foxit Reader"
         ]
     ],
-    "innosetup": true,
     "checkver": {
         "url": "https://www.foxitsoftware.com/downloads/downloadForm.php?retJson=1&product=Foxit-Reader&platform=Windows&language=English",
-        "jsonpath": "$.package_info",
-        "regex": "(?s)([\\d.]+)\".*\"down\":\\s*\"(?<dl>[^\"]+)\""
+        "jsonpath": "$.package_info.version.[0]"
     },
     "autoupdate": {
-        "url": "https://cdn01.foxitsoftware.com/$matchDl"
+        "url": "https://cdn01.foxitsoftware.com/pub/foxit/reader/desktop/win/$majorVersion.x/$majorVersion.$minorVersion/en_us/FoxitReader$majorVersion$minorVersion_Setup_Prom_IS.exe#/dl.7z"
     }
 }

--- a/bucket/foxit-reader.json
+++ b/bucket/foxit-reader.json
@@ -6,8 +6,8 @@
         "identifier": "Freeware",
         "url": "https://www.foxitsoftware.com/pdf-reader/eula.html"
     },
-    "url": "https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.0/B482F50BE0AE94322AED580B59D2C060/FoxitReader100_enu_Setup_Prom.exe",
-    "hash": "md5:b482f50be0ae94322aed580b59d2c060",
+    "url": "https://cdn01.foxitsoftware.com//pub/foxit/reader/desktop/win/10.x/10.0/en_us/FoxitReader100_Setup_Prom_IS.exe",
+    "hash": "eac0bf3cdb0b629381c61fe9c1767d310f8654e34e30f19145d3e88f816cac00",
     "bin": "FoxitReader.exe",
     "shortcuts": [
         [
@@ -18,13 +18,10 @@
     "innosetup": true,
     "checkver": {
         "url": "https://www.foxitsoftware.com/downloads/downloadForm.php?retJson=1&product=Foxit-Reader&platform=Windows&language=English",
-        "regex": "([\\d.]+).*\\\\/(?<dlver>[\\d\\.]+)\\\\/(?<md5hash>([a-fA-F0-9]{32}))\\\\/(?<dlname>.+\\.exe)"
+        "jsonpath": "$.package_info",
+        "regex": "(?s)([\\d.]+)\".*\"down\":\\s*\"(?<dl>[^\"]+)\""
     },
     "autoupdate": {
-        "url": "https://cdn01.foxitsoftware.com/product/reader/desktop/win/$matchDlver/$matchMd5hash/$matchDlname",
-        "hash": {
-            "url": "https://www.foxitsoftware.com/downloads/downloadForm.php?retJson=1&product=Foxit-Reader&platform=Windows&language=English",
-            "regex": "$md5"
-        }
+        "url": "https://cdn01.foxitsoftware.com/$matchDl"
     }
 }

